src/Resources/contao/classes/DataContainer.php

```diff
@@ -510,7 +514,19 @@ protected function row($strPalette=null)
 			}
 		}
 
-		$objWidget->wizard = $wizard;
+		if ($wizard != '')
```

Comment:
It hides the picker icon and spacing if the dcaPicker is not supported (due to permissions etc.)